### DISCUSSION TITLE
Multiple Camera and Audio Inputs with Overlays

### DISCUSSION THREAD

**Discussion by MIDIculous** [ant-media/Ant-Media-Server#6647] [Q&A]
Would anyone be able to offer any suggestion for getting multiple camera and/or audio inputs without the use of GStreamer? I'm creating an online teaching app where students need to see multiple camera angles and or audio signals. 

Any help would be appreciated.

**Comment by rahul7827** [comment]
Hello @MIDIculous 

This can be helpful to you. [Multitrack publish & Play](https://antmedia.io/docs/guides/publish-live-stream/multitrack-publish-and-play-with-ams/) OR Conference. I recommend you to have a birds eye of view of all documentation pages.
